### DISCUSSION TITLE
[@types/gulp-changed]: The deprecated function `compareSha1Digest()` replaced with `compareContents()`

### DISCUSSION THREAD

**Discussion by david-bojnansky** [DefinitelyTyped/DefinitelyTyped#58040] [Issues with a @types package]
Hello,

in `gulp-changed` v4.0.0 there was removed deprecated function `compareSha1Digest()`. On the other hand, in `@types/gulp-changed` there is missing `compareContents()` which replaced the deprecated `compareSha1Digest()`.

Thanks for fix.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready #58068
